Documentation/policy/language.rst

```diff
@@ -984,95 +979,11 @@ matching names are inserted irrespective of how the data is obtained. These IPs
 can be selected with ``toFQDN`` rules. DNS responses are cached within cilium
 agent respecting TTL.
 
-.. _DNS Proxy:
-
-DNS Proxy (preferred)
```

Comment:
The intent was just to remove the ` (preferred)` part of this title, because there's no preference between a single option.

Please restore this piece of the docs, thanks!